src/update_engine/download_action.cc

```diff
@@ -60,7 +60,15 @@ void DownloadAction::PerformAction() {
     processor_->ActionComplete(this, kActionCodeInstallDeviceOpenError);
     return;
   }
-
+  if (delta_performer_.get() &&
+      !delta_performer_->OpenKernel(
+          install_plan_.kernel_install_path.c_str())) {
+    LOG(ERROR) << "Unable to open kernel file "
+               << install_plan_.kernel_install_path.c_str();
```

Comment:
I don't think it is a style thing, it was just a simple copy/paste mistake in the original code from the above call to OpenKernel which does use a char*